### PR TITLE
BZ1827131 - Update LSO wording

### PR DESCRIPTION
**Pull request by bobfuru**
Minor formatting updates to LSO doc as described in [BZ1827131](https://bugzilla.redhat.com/show_bug.cgi?id=1827131).

### CODE REVIEW

**Comment by bobfuru** [comment]
@openshift/team-documentation PTAL

File: modules/persistent-storage-local-install.adoc [hunk 1]
```diff
@@ -59,39 +59,35 @@ Once finished, the Local Storage Operator will be listed in the *Installed Opera
...
-    sourceNamespace: openshift-marketplace
+apiVersion: operators.coreos.com/v1alpha2
+kind: OperatorGroup
+metadata:
+  name: local-operator-group
+  namespace: local-storage
+spec:
+  targetNamespaces:
+    - local-storage
+---
```
Comment (jeana-redhat) [inline modules/persistent-storage-local-install.adoc:76]: In [the preview build](https://bz1827131--ocpdocs.netlify.app/openshift-enterprise/latest/storage/persistent_storage/persistent-storage-local.html#local-storage-install_persistent-storage-local), this renders in green text and underlined (the same way a `...` would indicating trimmed content).

**Review by jeana-redhat — COMMENTED** [review]
Found an odd styling aberration, otherwise LGTM. LMK if you need approval when you get the style thing figured out :)

**Review by jeana-redhat — APPROVED** [review]
formatting fixed now :+1:

**Comment by bobfuru** [comment]
Thanks, @Jeana, for pointing out the `---` issue!

**Comment by bobfuru** [comment]
@qinpingli PTAL for QE ack.

**Comment by qinpingli** [comment]
@chao007 Please help take a look, thx!

**Pull request merged**